transcom/mymove
# B 18480 int

## [B-18480t](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A888864)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

PPM move is created by Customer from a location that is not services counseled
login in as TOO of origin location to approve move and approve services counseling service item
login as prime  to submit  payment request for counseling
login as TIO to review payment request
click review weights > click review shipment weights
verify estimated weight is now displaying

### Frontend


## Screenshots
![image](https://github.com/transcom/mymove/assets/136509766/33bd6f10-732f-4e75-88bc-e6b31b53bd0c)
